docs/import/add_disaster.js

```diff
@@ -0,0 +1,253 @@
+import {eeLegacyPathPrefix, gdEePathPrefix} from '../ee_paths.js';
+import {getFirestoreRoot} from '../firestore_document.js';
+
+export {enableWhenReady, SENTINEL_NEW_DISASTER_VALUE, toggleState};
+// Visible for testing
+export {
+  addDisaster,
+  createAssetPickers,
+  createOptionFrom,
+  disasters,
+  emptyCallback,
+  getAssetsFromEe,
+  stateAssets,
+  writeNewDisaster,
+};
+
+// Currently a map of disaster name to states. This pulls once on firebase
+// authentication and then makes local updates afterwards so we don't need to
+// wait on firebase writes to read new info.
+const disasters = new Map();
+
+// Map of state to list of known assets
+const stateAssets = new Map();
+
+const SENTINEL_OPTION_VALUE = '...';
+const SENTINEL_NEW_DISASTER_VALUE = 'NEW DISASTER';
+
+/**
+ * Populates the disaster picker with disasters from firestore, and enables
+ * the ability to add a new disaster.
+ * @return {Promise<firebase.firestore.QuerySnapshot>}
+ */
+function enableWhenReady() {
+  // enable (currently hidden) add disaster button now that firestore is ready.
```

Comment:
remove? And rename method?

Reply:
Sorry - did mean to leave as disabled and enable here.